packages/admin/cms-admin/src/vendors.d.ts

```diff
@@ -1,5 +1,6 @@
 /// <reference types="@comet/admin-theme" />
 
+// eslint-disable-next-line no-restricted-globals
```

Comment:
I could not get it to work, when removing the `React.` usage here.

When its removed, there is an error, that the `ref` prop is missing in `CustomInputProps`.
And when the two types are imported `import { type DetailedHTMLProps, type InputHTMLAttributes } from "react";` there is an error, that the added prop `webkitdirectory` is not found.

Do you have idea what's going on here? Can we leave the `React.` usage here in the vendor.d.ts file (in the typedefinitions `React.` usage seems common?)

Reply:
This should work:

```ts
/// <reference types="@comet/admin-theme" />

import { type HTMLAttributes } from "react";

declare module "react" {
    interface InputHTMLAttributes<T> extends HTMLAttributes<T> {
        // extends React's HTMLAttributes
        directory?: string;
        webkitdirectory?: string;
    }
}
```

Reply:
> Can we leave the React. usage here in the vendor.d.ts file (in the typedefinitions React. usage seems common?)

We don't need it in other .d.ts files 🤔 Maybe we should add explicit imports to all .d.ts files, otherwise probably the namespace is still used?

Reply:
@johnnyomair i found no other usage?

Reply:
I meant for instance here: https://github.com/vivid-planet/comet/pull/3376/files#diff-4673cf9aa84326b13c17dd7c9168e6d127c88afef4682665b97e4edc7652e277. Here you can use `ReactNode` without an import. In my editor it resolves to  `any`. Maybe we should add an import here?

Reply:
okay 👍 
imports added.